src/AddTodoForm.jsx

```diff
@@ -1,18 +1,25 @@
-const AddTodoForm = (props) => {
+import { useState } from 'react';
+
+const AddTodoForm = ({ onAddTodo }) => {
+  const [todoTitle, setTodoTitle] = useState('');
+
+  const handleTitleChange = (e) => {
+    const newTodoTitle = e.target.value;
+    setTodoTitle(newTodoTitle);
+  }
 
   const handleAddTodo = (e) => {
     e.preventDefault();
-    let todoTitle = e.target.title.value;
     console.log(todoTitle);
```

Comment:
I see that the todoTitle is still being logged here. Did you mean to leave this in? Definitely not a problem, but usually debug logs like this would be removed before a merge, so I just wanted to point it out.